<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCCarDialog</class>
 <widget class="QDialog" name="addCCarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add company car</string>
  </property>
  <widget class="QDialogButtonBox" name="addCCarButtonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_21">
    <item>
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_53">
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="licnr_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_61">
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comname_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_63">
      <property name="text">
       <string>Organization Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="orgnr_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_64">
      <property name="text">
       <string>Ensured by:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ensur_CC_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addCCarButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addCCarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCCarButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addCCarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
